lib/tasks/read-config.js

```diff
@@ -32,14 +32,23 @@ module.exports = Task.extend({
 
   _loadDotEnv: function() {
     var root         = this.root;
-    var dotEnvFilename = '.env.deploy.' + this.deployTarget;
+
+    var deployDotEnvFilename = '.env.deploy.' + this.deployTarget;
+    var deployDotEnvFilePath = path.join(root, deployDotEnvFilename);
+
+    var dotEnvFilename = '.env';
     var dotEnvFilePath = path.join(root, dotEnvFilename);
 
-    if (existsSync(dotEnvFilePath)) {
-      dotenv.load({
-        path: dotEnvFilename
-      });
-    }
+    // order is important here. vars defined in files loaded first
+    // will override files loaded after.
```

Comment:
I added this comment here because the behavior in dotenv seem backwards to me. I would expect overrides to be loaded last!